L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -94,7 +95,16 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
     # load the Stage 1 configuration
     process = customiseSimL1EmulatorForStage1(process)
     # set the Stage 1 heavy ions-specific parameters
-    process.simCaloStage1Digis.FirmwareVersion = cms.uint32(1)
+    # all of these should eventually end up in a GT
+    process.RCTConfigProducers.eicIsolationThreshold = cms.uint32(7)
```

Comment:
@mulhearn Two of the HLT breaks come from my commits - they can be fixed by adding "hasattr" guards on process.RCTConfigProducers here.